app/routes.js

```diff
@@ -16,6 +16,7 @@ exports.init = app => {
   app.get('/health', healthCheck);
 
   app.post('/signin', signInValidator, userController.signIn);
+  app.post('/users/sessions/invalidate_all', validateToken, userController.invalidateAllSessions);
```

Comment:
For routes it is preferable to use kebab-case instead o snake_case

Reply:
Ready!